src/VisualStudio/Core/Def/LanguageService/AbstractLanguageService`2.VsCodeWindowManager.cs

```diff
@@ -249,15 +250,17 @@ int IVsDocOutlineProvider.GetOutline(out IntPtr phwnd, out IOleCommandTarget? pp
                 var asyncListenerProvider = _languageService.Package.ComponentModel.GetService<IAsynchronousOperationListenerProvider>();
                 var asyncListener = asyncListenerProvider.GetListener(FeatureAttribute.DocumentOutline);
                 var editorAdaptersFactoryService = _languageService.Package.ComponentModel.GetService<IVsEditorAdaptersFactoryService>();
+                var workspace = _languageService.Workspace;
+                var navigationService = workspace.Services.GetRequiredService<IDocumentNavigationService>();
```

Comment:
not needed.  can be retrieved on demand from teh workspace passed into the view.